tensorboard/writer/BUILD

```diff
@@ -0,0 +1,42 @@
+# Description:
+# TensorBoard, a dashboard for investigating TensorFlow
+
+package(default_visibility = ["//tensorboard:internal"])
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+py_library(
+    name = "writer",
+    srcs = glob([
+        "*.py",
+    ]),
+    srcs_version = "PY2AND3",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//tensorboard:expect_numpy_installed",
+        "//tensorboard/compat/proto:protos_all_py_pb2",
+        "@org_pythonhosted_six",
+    ],
+)
+
+
+py_test(
+    name = "event_file_writer_test",
+    size = "small",
+    srcs = ["event_file_writer_test.py"],
+    main = "event_file_writer_test.py",
+    srcs_version = "PY2AND3",
+    deps = [
+        ":event_file_writer",
```

Comment:
I would first try just making this

```
    deps = [
        ":writer",
        "//tensorboard:expect_tensorflow_installed",
    ]
```

and then test to see what breaks with

```
bazel test //tensorboard/... --test_output=errors
```

Reply:
nothing breaks with the test.  ( with `deps = [":writer",]`  )
Does that mean I can remove other deps safely?

Reply:
That should do it.